versions/1/kibana/spec.yml

```diff
@@ -74,6 +74,15 @@
         type: file
         contentMediaType: "application/json"
         pattern: '^.+\.json$'
+    - description: Folder containing CSP rule templates
+      type: folder
+      name: "csp_rule_template"
+      required: false
+      contents:
+      - description: An individual CSP rule template file for the cloud security posture management solution
+        type: file
+        contentMediaType: "application/json"
+        pattern: '^.+\.json$'
```

Comment:
A question From my lack of knowledge on these features :slightly_smiling_face:

I see there is already a directory for `security_rule`s, how are these csp rule templates different to the security rules above? Are they installed using the same APIs?

I am wondering if we could use the same `security_rule` directory for both kinds of files if they are handled the same way on installation, or use a subdirectory in `security_rule` for these CSP templates if they are different assets, but belong to the same feature.

Reply:
Hey @jsoriano, Thanks for reviewing!
This option (using the existing `security-rule` asset) did came up in the [issue](https://github.com/elastic/package-spec/issues/263), but since they are completely unrelated in Kibana (all `security-rule`s are part of the detection engine, `csp-rule-template`s are relates to the could security posture plugin) and require different installation process (usage of [hooks](https://github.com/elastic/kibana/issues/124149#issuecomment-1032762790)) it seemed like everyone agree they should be separate types of assets.

Reply:
Ok, makes sense, thanks!